src/plugins/sorting-task/SortingTaskNode/LearningItems/styles.css

```diff
@@ -0,0 +1,11 @@
+.learning-items{
```

Comment:
We should use `.scss` and nest these selectors like we do in the other components for better structure.

Reply:
I’m not 100% sure about nesting. In the ideal case, we don’t need any super-specific selectors, thus we don’t nesting as well. In some cases it might be necessary to have slightly more complex selectors, but these should be a rare exception. Nesting by default means having higher-specificity selectors by default.

Reply:
I agree that we should keep it as simple as possible with selectors. Maybe only nest 2-3 levels deep when we see that its necessary otherwise we could end up with something like this [CSS Selector Nightmare](http://thesassway.com/beginner/the-inception-rule)